src/components/intro/index.tsx

```diff
@@ -66,6 +70,22 @@ export default class Intro extends Component<Props, State> {
     this.fileInput!.click();
   }
 
+  @bind
+  private onBeforeInstallPrompt(e: Event) {
+    this.setState({ canInstall: true });
+    this.bipEvent = e as BeforeInstallPromptEvent;
```

Comment:
If you wanna ship your own install UI, you need to `preventDefault()` here.

Reply:
Oops, I had that in onInstallClick. Fixed.